netdata.spec.in

```diff
@@ -18,7 +18,7 @@
 Summary:	Real-time performance monitoring, done right
 Name:		@PACKAGE_NAME@
 Version:	@PACKAGE_RPM_VERSION@
-Release:	@PACKAGE_RPM_RELEASE@%{?release_suffix}%{?dist}
+Release:	1%{?dist}
```

Comment:
This will fix https://github.com/firehol/netdata/pull/728#issuecomment-235645337